Plugins/Detray/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ add_library(
     src/DetrayConverter.cpp
     src/DetrayGeometryConverter.cpp
     src/DetrayMaterialConverter.cpp
+    src/DetraySurfaceGridsConverter.cpp
 )
 
 add_dependencies(ActsPluginDetray detray::core covfie::core vecmem::core)
```

Comment:
```detray::utils``` does not exist anymore: To get specific detector metadata types (except the default one which is in ```detray::core```) there is ```detray::detectors``` now and to get extended validation tools (probably only for the executables where we do the testing) there is ```detray::test_utils```

Reply:
Not using `detray::utils` here or?